### ISSUE TITLE
ascii in filenames?

### ISSUE THREAD

**Issue by kberry** [TeXUsersGroup/TUG-Conference#22]
Just have to ask, sorry -- would it cause trouble to keep the names of files committed to the repo to 7-bit ASCII? There are a few (speaker names) in UTF-8. If you have automation set up to make use of that fact, that's fine, don't bother to change. But if it's just arbitrary, sticking to ascii would simplify a lot for me.

(list attached)

[nonascii.txt](https://github.com/TeXUsersGroup/TUG-Conference/files/9538831/nonascii.txt)

**Comment by pauloney** [comment]
Karl,

Are you talking, for example, about:


  TUG-Conference/assets/served/Tereza_Vrabcová-TUG2022-vrabcova-markdown-slides.pdf

that is, the files in subdirectories of "TUG-Conference/assets/served/"?

These are NOT written by anyone. The script does the whole job. There is no
need for it to be at
GitHub, it was just convenience of writing "commit -a" in the beginning,
but we can, in fact remove
the entire dir from GitHub.

Paulo Ney

**Comment by kberry** [comment]
TUG-Conference/assets/served

No, the files I noticed were under assets/speakers/. Photos, like
assets/speakers/Tereza-Vrabcov<...>.png (sorry, can't copy/paste utf8
here). --thanks, karl.

**Comment by pauloney** [comment]
On Fri, Sep 9, 2022 at 2:36 PM kberry ***@***.***> wrote:

> TUG-Conference/assets/served
>
> No, the files I noticed were under assets/speakers/. Photos, like
> assets/speakers/Tereza-Vrabcov<...>.png (sorry, can't copy/paste utf8
> here). --thanks, karl.
>


We are changing that. In the final version the files will be named by th
"name-tag" on the DB and that is ASCII. The version there right now
uses UTF-8 because it was done by hand.

Paulo Ney

**Comment by yoniros** [comment]
> The script does the whole job. There is no need for it to be at GitHub, it was just convenience of writing "commit -a" in the beginning, but we can, in fact remove the entire dir from GitHub.

@pauloney - just a quick comment: the directory `assets/served` is in fact **not** in GitHub, and never was. (You can see it listed in [.gitignore](https://github.com/TeXUsersGroup/TUG-Conference/blame/main/.gitignore#L4)).

**Comment by pauloney** [comment]
Yes, I saw that. The one his is worries about is the one with the photos
that will end up with ASCII names after we automate the generation of that
page. There will be files using UTF-8 but not in their names.

PN
